kernel/enter.c

```diff
@@ -170,6 +169,16 @@ static int _copy_host_etc_files()
     return ret;
 }
 
+static int _copy_host_etc_files()
+{
+    int ret = 0;
+    ECHECK(_copy_host_etc_file("/etc/resolv.conf"));
+    ECHECK(_copy_host_etc_file("/etc/hosts"));
+    ECHECK(_copy_host_etc_file("/etc/hostname"));
```

Comment:
hostname is part of the our config.json:

<!--StartFragment-->

Hostname | string | The default hostname exposed to application
-- | -- | --


<!--EndFragment-->

Not sure how this may work in conjunction with that.

Reply:
Fixed.